### ISSUE TITLE
Unyson Builder Disappears & Pages lose information and formatting

### ISSUE THREAD

**Issue by melindahug** [ThemeFuse/Unyson#4348]
I have Unyson 2.7.28 on my Wordpress site. When it updates to 2.7.31, the page editor disappears and the pages lose their info and formatting. The only way to preserve my site is to revert to the saved copy from before the update. If I click on "View Details," it says Plugin not Found. If I click on Themefuse, it goes to a broken web page. I do not have auto updates enabled, but it seems it "updates" on its own and breaks overnight after I revert to the old site copy. Does anyone know what's going on? Or know how I can fix this issue?
<img width="1461" alt="Unyson" src="https://github.com/ThemeFuse/Unyson/assets/166972218/4f5fa772-5fc9-4ba4-a734-b696cfb124f5">

**Comment by ManySolutions** [comment]
I don't understand. plesk is advising i deactivate the plug in. once done it breaks the site. Nothing is available in information and the page to submit ticket to themefuse is down. I feel like this is the end of unyson? I have to go back and reconsider all of my themes.

**Comment by CreativeManner** [comment]
Hey @melindahug. I use the plugin on a few of my clients' sites. Let me know if you need help.

**Comment by melindahug** [comment]
@ManySolutions Yes, I tried to deactivate and delete Unyson, and it crashes the whole site. Unyson is dead apparently and Brizy is the replacement from what I understand. I think we have to rebuild the pages with Brizy once installed. I haven't done anything yet because I don't know if installing Brizy on top of Unyson will mess everything up.

**Comment by melindahug** [comment]
Hey @zenics. Is Unyson acting up on any of your client's sites? The pages that break on my site when Unyson disappears are videos and booking calendars. Do you know anything about installing Brizy on top of Unyson and rebuilding pages with Brizy?
